frontend/src/AudioAnnotator/Region.js

```diff
@@ -109,12 +109,13 @@ class Region extends Component<RegionProps, RegionState> {
             <button
               className="btn-simple fa fa-play-circle"
               onClick={this.playAnnotation}
-            ></button>
+          ></button>
```

Comment:
I'm guessing you have an automated tool that changed the tabulation ? I'm a bit perplexed why it prefers it this way because I would rather have the previous alignment :thinking: